### DISCUSSION TITLE
Add Camera Selector to Demos

### DISCUSSION THREAD

**Discussion by nskaiser** [vladmandic/human#227] [Ideas]
Hi Vlad,  If possible and time allows, can you add a camera selector to your online demos? I often like to test things using a virtual camera (e.g SplitCam) to feed a video through and Chrome seems to have a mind of its own when choosing which camera to use, despite the defaults.  Thanks.

**Comment by vladmandic** [comment]
i think its an overkill to add it to demos as its not a frequent use-case (and demos are just that - demos to showcase library - not meant to be a fully featured applications), but its trivial to implement on your side  

see <https://github.com/vladmandic/human/blob/5de30e6233ef9fc5616a14b3b648ee0b84d8c6c7/demo/index.js#L393-L399>

**Comment by nskaiser** [comment]
Thanks, makes sense.  Yes, we have these in our apps. Just thought it might be of use in yours. Thanks anyway.
